### PR TITLE
Front End Plugin Card File Attachment

### PR DESCRIPTION
**Pull request by tejasj1718**
### What this PR does (required):
- Implements the FE Plugin Card File Attachment
- Uses storybook to view the changes in isolation.
- Adds mock file data and another File Upload component.

### Screenshots / Videos (front-end only):
- The File Attachment Plugin. Will be placed on the detailed card.
PRfeFileAttach

### Any information needed to test this feature (required):
- run the command `npm run storybook`
- Test the component.

### Any issues with the current functionality (optional):
- Not hooked up to the backend, uses mock data.
- Overlap between image upload and file upload component, could use a refactor.

### CODE REVIEW

File: client/src/components/plugins/FileAttach/FileDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    updatedFiles.splice(fileIndex, 1);
+    setFiles(updatedFiles);
+    setOpen(false);
+  };
+
+  const classes = useStyles();
+
+  return (
+    <Grid item>
+      <a href={file.url} download>
```
Comment (ezeilo-su) [inline client/src/components/plugins/FileAttach/FileDisplay.tsx:40]: Please use MUI `Link` component for `a` element

File: client/src/components/plugins/FileAttach/FileAttach.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  const handleClick = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleUpload: IDropzoneProps['onSubmit'] = () => {
+    //This function will upload files and set the files state.
```
Comment (ezeilo-su) [inline client/src/components/plugins/FileAttach/FileAttach.tsx:32]: Please be sure to remove this comment :smile:

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Nice work, @tejasj1718. A few minor comments, though. :smiley:

**Review by moffatethan — APPROVED** [review]
Good work!

**Pull request merged**